spec/tapioca/compilers/dsl/active_record_enum_spec.rb

```diff
@@ -87,6 +84,82 @@ def archived?; end
       assert_equal(rbi_for(content), expected)
     end
 
+    it("generates RBI file for classes with an enum attribute with hash values") do
+      content = <<~RUBY
+        class Conversation < ActiveRecord::Base
+          enum status: { active: 0, archived: 1 }
```

Comment:
Can we change the values here to be `String`s instead. It would be good to test the case where we discover a single non-`Integer` type.